java/RSPEC-2757.java

```diff
@@ -0,0 +1,7 @@
+class InvertedAugAssign {
+    void test() {
+        int a = 1;
+        a =+ 2;
+        a =- 3;
```

Comment:
RSPEC-2757: Don't repeat Unary operators
<!-- lookout footnote separator -->
_Comment made by the analyzer sonarcheck_analyzer._